blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+* Balanced Cut (clustering)
+* Spectral Clustering (clustering) 
+* HITS (hubs vs authorities analytics)
+* Leiden (community detection)
+* Katz Centrality 
+* Betweenness Centrality  
+
+All above algorithms are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**. 
+
+## Prerequisites
+
+To follow the tutorial, please install:
+- [Docker](https://docs.docker.com/get-docker/) - needed to run `mage-cugraph` image
+- [Jupyter](https://jupyter.org/install) - using jupyter-notebook you can write  `CSV` importing and graph analytics in one file
+- [GQLAlchemy](https://pypi.org/project/gqlalchemy/) - to connect to Memgraph with Python
+- [Memgraph Lab](https://memgraph.com/lab) - a GUI tool we use to to visualize graphs
+
+
+### Memgraph with Docker
+We use *[Docker](https://www.docker.com/)* because the Memgraph database is a native Linux application and cannot be installed on Windows and macOS. But for this part, we will use **Docker image** of  `mage-cugraph`. [Here](https://memgraph.com/docs/mage) you can check how to pull the image, and set everything up before running the **Memgraph-cuGraph** container.
+
+
+Before running `mage-cugraph` Docker image, you should [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) **[jupyter-memgraph-tutorials](https://github.com/memgraph/jupyter-memgraph-tutorials)** GitHub repo and position yourself inside `jupyter-memgraph-tutorials/cugraph-analytics` folder. We will do one "hack" here for later. Since we are working with *Docker* and we will read our **CSV** files, **Memgraph** needs to have access to such files. But they are currently on our machine and need to be inside *Docker*. That's we will create **Docker** volume mounting our current `data/facebook_clean_data/` folder to `/samples` folder inside the Docker container. Inside `/samples` folder, *Memgraph* will find our **CSV** files.
+
+After you cloned the repo and positioned yourself in it, now we can start the `mage-cugraph` image:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 --volume ./data/facebook_clean_data/:/samples mage-cugraph
+```
+
+
+### Jupyter notebook
+After you have Memgraph running, you should install **Jupyter**. For this tutorial, we used **jupyter-lab** and you can install it with the following command:
+```
+pip install jupyterlab
+```
+
+Now you can go and download **[jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics)** we use in our tutorial. Note, in a repository folder `cugraph-analytics/data` there are `CSV` files containing the full dataset which we will use later in the tutorial.
```

Comment:
Next, download [jupyter-notebook].... . As mentioned before, CSV files holding the dataset we will use in the tutorial are located in the repository folder `cugraph-analytics/data` .

[when giving instructions avoid using to many words -> now you should install this == install this, now you can go and upload that = upload that]

IMHO, instructions need to be clear and short, and we can fluff the text around them.

Reply:
You are right, I agree that instructions should be clear. 👍🏻